tests/rule_based_profiler/parameter_builder/test_regex_pattern_string_parameter_builder.py

```diff
@@ -312,7 +314,7 @@ def test_regex_pattern_string_parameter_builder_bobby_no_match(
                 r"/(?:[A-Fa-f0-9]){0,4}(?: ?:? ?(?:[A-Fa-f0-9]){0,4}){0,7}/": 0,
                 r"\b[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}-[0-5][0-9a-fA-F]{3}-[089ab][0-9a-fA-F]{3}-\b[0-9a-fA-F]{12}\b ": 0,
             },
-            "threshold": 0.9,
+            "success_ratio": 0.0,
```

Comment:
@Shinnnyshinshin This is odd -- how come so low?  Is it because no regex matches?

Reply:
Yes, this is a test for "no-match" :)